### PR TITLE
Feat: Format duration ms to duration

### PR DESCRIPTION
**Pull request by developerr-ayush**
[Open in Gitpod](https://gitpod.io/from-referrer/)&nbsp;[know more](https://www.gitpod.io/docs/pull-requests/)

### Describe your change:

- [x] Add an algorithm?
- [ ] Fix a bug or typo in an existing algorithm?
- [ ] Documentation change?

### Checklist:

- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Javascript/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized.
- [x] I know that pull requests will not be merged if they fail the automated tests.
- [x] This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.
- [x] All new JavaScript files are placed inside an existing directory.
- [x] All filenames should use the UpperCamelCase (PascalCase) style. There should be no spaces in filenames.
      **Example:**`UserProfile.js` is allowed but `userprofile.js`,`Userprofile.js`,`user-Profile.js`,`userProfile.js` are not
- [x] All new algorithms have a URL in their comments that points to Wikipedia or another similar explanation.
- [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by developerr-ayush** [comment]
1. hey can you give me hectoberfest label?
2. do you have any resource for understanding forking if you see above then there are lots of commits visible and that commits are from previous merge request although it doesn't affect but how can I sort this?
3. i have tried the rebase thing but yet didn't worked.

Thanks

File: Conversions/FormatDuration.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/**
+ * Formats a duration in milliseconds into a human-readable string.
+ *
+ * @param {number} ms - The duration in milliseconds (can be negative).
+ * @returns {string} A formatted string representing the duration.
+ */
+const FormatDuration = (ms) => {
+  if (ms < 0) ms = -ms
```
Comment (appgurueu) [inline Conversions/FormatDuration.js:8]: Why is this useful or reasonable? I'd remove it altogether and just not allow negative durations.

**Review by appgurueu — APPROVED** [review]
Code is fine, has tests and docs. I'm not sure whether we can consider this algorithmically interesting enough, though. The string formatting logic here is pretty straightforward.

**Review by appgurueu — COMMENTED** [review]
Why did you add a `msToDuration` function here?

**Comment by developerr-ayush** [comment]
@appgurueu it is a related function to the same which converts ms to some other string that's why I have added that here in this pr

**Comment by appgurueu** [comment]
I don't really see a point in having two functions around which do similar things.

Still, the main question persists: What does this - algorithmically - teach a reader?

**Comment by developerr-ayush** [comment]
so we have video playforms in that the second function can be used to show the live duration and the other one is kinda hard to tell what will be helpful from that

**Comment by developerr-ayush** [comment]
i have faced this one i have to make an timer for OTP for that I have to take 30 sec, 5 min , 30 min. so in this case I have made this algo for first function which has names in output. second one was a video player to show live duration counter as I was getting data as ms

**Comment by raklaptudirm** [comment]
That is not the point @developerr-ayush. It maybe a useful modular piece of code, but remember that this repository is not supposed to be used as a module. All the code added here should provide some educational value to the reader, and therefore useful but trivial algorithms won't be accepted.

**Comment by developerr-ayush** [comment]
Okay i think its useless then

**Pull request closed without merging**